about.html

```diff
@@ -0,0 +1,154 @@
+                <span  class="section-header-second-title-year" id="date"></span>
+            </h2>
+            <p class="section-header-text-content lorem">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus
+                usto, maxime nulla, usto, maxime nulla, iusto, maxime nulla, nesciunt
+                reprehenderit.Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Temporibus usto, maxime nulla, usto, maxime nulla, iusto, maxime
+                nulla, nesciunt reprehenderit.Temporibus usto, maxime nulla, usto,
+                maxime nulla, iusto, maxime nulla, nesciunt reprehenderit.Temporibus
+                usto, maxime nulla, usto, maxime nulla., iusto, maxime
+            </p>
+            <div class="section-header-second-content">
+                <small class="section-header-info">
+                    Please contact us through Email for any further questions about us.
+                </small>
+                <small><br /><strong class="section-header-strong-text">tofindoutyeah@gmail.com</strong></small>
+            </div>
+        </section>
+    </section>
+    <main>
+        <div class="main-section-info text-center">
+            <h2 class="main-section-title">
+                ©2021 Yve's Konditorei. All Rights Reserved.
+            </h2>
+            <p class="footer-top-underline display-flex-center">
+                <span class="footer-top-underline-span"></span>
+            </p>
+            <p class="main-section-text">
+                Altendettelsauer Str 1
+                Petersaurach, 91580
+                Germany
+            </p>
+            <div class="main-section-info-logo">
+                <img class="main-section-info-logo-image" src="images/logo.jpeg" alt="LOGO" />
+            </div>
+        </div>
+    </main>
+    <article class="pictures">
+        <div class="main-section-info">
+            <h2 class="main-section-title">See the new additions to our menu</h2>
+            <p class="footer-top-underline display-flex-center">
+                <span class="footer-top-underline-span"></span>
+            </p>
+            <p class="main-section-text">
+                Take a look at the most requested desserts in our carte.
+                Ansbash, Petersaurach, Germany ©2021
+            </p>
+            <div class="images">
+                <div class="image-one">
+                    <p class="image-title">2019</p>
+                    <span class="image-text">Yve's Konditorei recognoized as the most exquisite Confectionery in Bavaria's contest on 2019</span>
+                </div>
+                <div class="image-two">
+                    <p class="image-title">2020</p>
+                    <span class="image-text">Konditorei recognoized as the most exquisite Confectionery in Munich's contest on 2020</span>
+                </div>
+            </div>
+        </div>
+    </article>
+    <footer>
```

Comment:
- [ ] Both pages should contain the same navbar and footer. For desktop version footer of the `home` page and `about` page are not similar. There is a dark background colour for the footer in the desktop version. Please try to fix it.

Home page footer:- 
![image](https://user-images.githubusercontent.com/53019916/125175347-1fad1b00-e1e9-11eb-932b-77976ff9494c.png)

About page footer:- 
![image](https://user-images.githubusercontent.com/53019916/125175376-453a2480-e1e9-11eb-98ff-86a35a5e7041.png)

Reply:
I fixed the errors